### ISSUE TITLE
Table didn't reflect changes of unselectable prop when passed array changes dynamically.

### ISSUE THREAD

**Issue by anujemc** [AllenFang/react-bootstrap-table#670]
I am currently using react-bootstrap-table v2.4.0 and I have face problem that table didn't reflect changes of unselectable prop when passed array changes dynamically.
I have also checked issue (https://github.com/AllenFang/react-bootstrap-table/issues/550) related to this  but according to that thread it has been fixed in v2.3.8 .

**use case -** 
first point here is I am using react-bootstrap table in a Modal which pops up on button click .
In that table we have some modules and its status , when click on button('Start Trial') , the status got updated to some other status like 'In progress' and also unselectable array has been updated along with new key value .
Means status related changes reflected in table but checkbox is still in active state and not get disabled even though unselectable array has been updated.

Above mentioned checkbox only get disabled when I close that modal and reopen that modal again .

Tried to force update but didn't work
```
componentDidUpdate () {
       setTimeout(() => {
           this.refs.BootstrapTable.forceUpdate();
       }, 500);
   } 
```
![starttrial](https://cloud.githubusercontent.com/assets/19282193/18125097/86f369a0-6f92-11e6-8ac2-a7ed765756a1.PNG)

A quick response is appreciated , Thanks !

**Comment by anujemc** [comment]
@AllenFang one more observation I have noticed that , in that case when you check 'select all' , react-bootstrap-table exclude that entry which has been added recently with changed status , also not allow to select that entry using clicking on row , so in my opinion problem happened in the view part only so that checkbox not disabled properly because logic works perfectly fine .

**Comment by anujemc** [comment]
I will try to reproduce on some online editor, but It may take some time .

**Comment by AllenFang** [comment]
@anujemc, that's ok. thank you :) I'll fix it quickly

**Comment by AllenFang** [comment]
@anujemc, fixed on ```v2.5.1```.
